### PR TITLE
docs: update publishing-guidelines.md

### PR DESCRIPTION
**Pull request by anfibiacreativa**
Change suggestion for requirement for running the pipeline on internal operations

### CODE REVIEW

File: docs/development-guidelines/operational-guidelines.md [hunk 1]
```diff
@@ -83,10 +83,21 @@ GitHub repositories should be the central place for discussion of issues. When t
 
 [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) are a great tool to create backlogs, label issues, prioritize, groom and distribute task. Avoid using Teams or any other tool for that purpose, particularly when teams are distributed. Keeping all information centralized in the repo or around it, helps keeping teams aligned, focused and organized.
 
+> [!IMPORTANT]
+> We require template owners to abide by the integration guidelines for templates that are already published, to minimize the risk of regressions.
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]:
```suggestion
> We require template owners to abide by the integration guidelines for all published templates to minimize the risk of regressions.
```

File: docs/development-guidelines/operational-guidelines.md [hunk 2]
```diff
@@ -83,10 +83,21 @@ GitHub repositories should be the central place for discussion of issues. When t
...
 
 ## Working on an issue
 
-When working on a task, be it a new feature or a bugfix, always open a related issue and assign it to a member of the team. Offer regular status referencing the issue. 
+When working on a task, be it a new feature or a bugfix,** always open a related issue and assign it to a member of the team. Offer regular status referencing the issue. **
+
+All development must be be done in feature branches.
+
+> [!IMPORTANT]
+> When the template is already published the [https://github.com/microsoft/template-validation-action](https://github.com/microsoft/template-validation-action) must be configured to run when opening a pull-request against `main` (or the base branch). No pull-request should be approved and merged without passing the validation.
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]:
```suggestion
> When the template is already published the [https://github.com/microsoft/template-validation-action](https://github.com/microsoft/template-validation-action), it must be configured to run when opening a pull-request against `main` (or the base branch). No pull-request should be approved and merged without passing the validation.
```

File: docs/development-guidelines/operational-guidelines.md [hunk 3]
```diff
@@ -96,4 +107,4 @@ A team must agree to a roadmap mapping to one or several [milestones](https://do
 
 Once agreed to, a code freeze is generally non-negotiable, though certain tasks can be spilled or descoped, and assigned to a post delivery milestone.
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md:108]:
```suggestion
Once agreed to, a code freeze is generally non-negotiable, though certain tasks can be split or descoped, and assigned to a post-delivery milestone.
```

**Review by kristenwomack — APPROVED** [review]
Left a few suggested edits.

File: docs/development-guidelines/operational-guidelines.md (hunk 3 above)
Comment (anfibiacreativa) [reply docs/development-guidelines/operational-guidelines.md:108]: Split has a different meaning, though. Spillover is a scrum term for when you have to move uncompleted tasks from one sprint to another.

**Review by kristenwomack — COMMENTED** [review]
LGTM

**Pull request merged**